src/Command/CreateAdminCommand.php

```diff
@@ -53,6 +59,17 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ['ROLE_ADMIN']
         ));
 
+        $question = new ConfirmationQuestion('Allow for sending anonymous usage statistic? (y/n) ', true);
+        $answer = $this
+            ->getHelper('question')
+            ->ask($input, $output, $question);
+
+        $this->bus->dispatch(new ChangeConfig([
+            'telemetry' => $answer ? 'enabled' : 'disabled',
+        ]));
```

Comment:
We need to check if instance id file is already generated. We cannot ask about this option every time.